src/SOS/Strike/strike.cpp

```diff
@@ -3144,7 +3140,7 @@ DECLARE_API(DumpVC)
     EnableDMLHolder dmlHolder(dml);
     if (nArg!=2)
     {
-        ExtOut("Usage: !DumpVC <Method Table> <Value object start addr>\n");
+        ExtOut("Usage: %sdumpvc <Method Table> <Value object start addr>\n", SOSPrefix);
```

Comment:
don't find this variant particularly readable, but don't find it any worse I guess